docs/UserGuide.md

```diff
@@ -102,7 +102,7 @@ Format: `list`
 
 Edits an existing item in the Inventory book.
 
-Format: ` edit INDEX n/NAME q/QUANTITY s/SUPPLIER [t/TAG]…​`
+Format: ` edit INDEX n/NAME q/QUANTITY s/SUPPLIER [max/MAX_QUANTITY] [t/TAG]…​`
```

Comment:
I think all the arguments, except INDEX, should be optional here

Reply:
Fixed! thanks for pointing it out!